metadata/boundaries/zip-code-areas

```diff
@@ -0,0 +1,83 @@
+# Title
+
+Utah ZIP Code Areas
+
+## ID
+
+1d4cbfba-75da-4d7c-a9bb-b32902b0af6d
+
+## Brief Summary
+
+The estimated geographic extents for all ZIP codes in Utah, excluding those that are PO Box only.
+
+## Summary
+
+This dataset includes the approximate geographic boundaries of every five-digit Zone Improvement Program (ZIP) code in Utah. ZIP codes are approximated based on the locations of known mailing addresses and are represented as polygons.
+
+## Description
+
+### What is the dataset?
+
+ZIP codes were first introduced by the United State Postal Service (USPS) in 1963 to improve mail delivery. The first three digits of the five-digit code refer to the Sectional Center Facility (SCF), or the central processing facility for a given region's mail. The last two digits provide a more specific locale that the mail will be delivered to.
+
+### What is the purpose of the dataset?
+
+This dataset was developed to provide a reference for assigning ZIP codes to geocoded features in the [Utah Roads](https://gis.utah.gov/products/sgid/transportation/road-centerlines/) and [Utah Address Points](https://gis.utah.gov/products/sgid/location/address-points/) datasets. Utah ZIP Code Areas have also been used in a variety of analyses; for example, one such analysis maps housing affordability in Utah by ZIP code. You can read about how ZIP codes were used to quantify housing data on our [blog](https://gis.utah.gov/blog/2024-05-17-utah-housing-affordability-map/).
+
+This dataset should not be used to determine the definitive zip code for addressing mail that will be sent through the US Postal Service.  The [USPS website](https://tools.usps.com/zip-code-lookup.htm) should be used for this purpose.
+
+### What does the dataset represent?
+
+Each polygon in this dataset represents the approximate geographic area of a given ZIP code and includes information such as the city and county the code lies within.
+
+### How was the dataset created?
+
+UGRC produced this dataset from multiple data sources using a combination of methods. Editing methods included:
+
+- Using local ZIP code polygon data obtained from counties and cities.
+- Identifying place locations (cities, towns, populated areas, etc.) from the postal service website and address information system (AIS) and building ZIP code boundaries around these places.
+- Using the 2000 Census TIGER line file’s ZIP code attributes.
+
+<!--- I've got a list of updates for this dataset, but have been unsuccessful in finding how this dataset is updated or what needs to happen in order to justify an update. Will be gathering more information on this. -->
```

Comment:
@gregbunce

Reply:
we currently don't have an update schedule for this dataset. In general, it's a very challenging dataset to maintain. There is no defined steward, but once our office was created, it began being used. As of now, we work with the locals when we here there's an issue and make adjustments ad hoc. I believe we're at the junction with this dataset where we either: (a) attempt to work with the post office (which they may or may not be willing to do) to incorporate changes or (b) look for a source (such as [esri's layer](https://hub.arcgis.com/datasets/d6f7ee6129e241cc9b6f75978e47128b/explore)) that has more regular data updates.

Reply:
Oh, gotcha. In that case, I'll keep the wording in the metadata vague and include a link for people to reach out to us if they have further questions.